.github/ISSUE_TEMPLATE/bugfix.yml

```
@@ -0,0 +1,392 @@
+      placeholder: "Login form validation fails with special characters"
+    validations:
+      required: true
+
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Bug Description
+      description: |
+        **What is the current problematic behavior?** Be specific and detailed.
+      placeholder: |
+        When users enter passwords containing special characters (!, @, #, $), 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)